pkg/controller/dataimportcron-conditions.go

```diff
@@ -36,11 +34,11 @@ const (
 
 func updateDataImportCronCondition(cron *cdiv1.DataImportCron, conditionType cdiv1.DataImportCronConditionType, status corev1.ConditionStatus, message, reason string) {
 	if conditionType == cdiv1.DataImportCronUpToDate {
-		gaugeVal := float64(0)
+		gaugeVal := 0
 		if status != corev1.ConditionTrue {
 			gaugeVal = 1
 		}
-		DataImportCronOutdatedGauge.With(getPrometheusCronLabels(types.NamespacedName{Namespace: cron.Namespace, Name: cron.Name})).Set(gaugeVal)
+		metrics.SetDataImportCronOutdatedValue(cron.Namespace, cron.Name, gaugeVal)
 	}
```

Comment:
instead of using a magic number value, update the function to be more meaning full like:
`func SetDataImportCronOutdated(namespace string, name string, isOutdated bool)`

and all of this would become:

```
if conditionType == cdiv1.DataImportCronUpToDate {
  metrics.SetDataImportCronOutdated(cron.Namespace, cron.Name, status != corev1.ConditionTrue)
}
```

Reply:
+1

Reply:
I had to update as is, since `Set` get float64 not bool:
```
func SetDataImportCronOutdated(namespace string, name string, isOutdated bool) {
	labelString := strings.ToLower(namespace + "/" + name)
	var outdatedValue float64
	if isOutdated {
		outdatedValue = 1.0 // true is represented as 1.0
	} else {
		outdatedValue = 0.0 // false is represented as 0.0
	}
	dataImportCronOutdated.WithLabelValues(labelString).Set(outdatedValue)
}

```